samples/application-conf/application.properties

```
@@ -5,6 +5,12 @@
 # Defaults to "false", to enable, set to "true"
 #runzTests=true
 
+# flag to convert TAZ EDT Unit test results to JUnit
+# The test results are converted to Junit by the sample style sheets that are provided in the installation_path/taz_samples/xsl directory of the Early Development Testing for z/OS client.
+# the location of the style sheet should be mentioned in the build-conf/TazUnitTest.properties
+# Defaults to "false", to enable, set to "true"
+#tazunittest_resultsConvtoJunit=false
```

Let's remove the comment ... 
```suggestion
tazunittest_resultsConvtoJunit=false
```